modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -52,7 +52,7 @@
     public static final String STRING_ENUMS = "stringEnums";
     public static final String STRING_ENUMS_DESC = "Generate string enums instead of objects for enum values.";
 
-    protected String ngVersion = "7.0.0";
+    protected String ngVersion = "8.0.0";
```

Comment:
please revert the default version to `7.0.0` to avoid a breaking change and have this feature released in the next release
```suggestion
    protected String ngVersion = "7.0.0";
```